### PR TITLE
Invocations list

### PR DESCRIPTION
**Pull request by KateVuilova**

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -28,9 +28,7 @@ class Header extends Component {
                         </div>
                         <div className="divider" />
                         <div className="filter-menu">
-                            {this.props.location.pathname === "/dashboard/24" ||
-                            this.props.location.pathname === "/dashboard/week" ||
-                            this.props.location.pathname === "/dashboard/month" ? (
+                            {this.props.location.pathname.includes("dashboard") ? (
```
Comment (mrmckeb) [inline src/components/Header/Header.js]: There's actually another way you can do this with React Router - by using a route. This is a design choice, but you may want to consider it as it makes the code quite readable to some people.

File: src/components/List/List.js [hunk 2]
```diff
@@ -39,13 +40,31 @@ class List extends Component {
...
+                        <td
+                            key={index}
+                            className={
+                                column.className
+                                    ? value[column.invalidParam]
+                                      ? `${column.className} ${column.invalidClassName}`
+                                      : column.className
+                                    : ""
+                            }
+                        >
```
Comment (mrmckeb) [inline src/components/List/List.js]: If only the content is changing, this `td` wrapper should be shared somehow.

File: src/containers/App/App.js [hunk 3]
```diff
@@ -79,7 +80,17 @@ class App extends Component {
                     <Sidebar />
                     <div className="main">
                         <Switch>
-                            <Route path="/dashboard/:dashboardType" render={() => <Dashboard />} />
+                            <Route
+                                path={
+                                    "/dashboard/:dashboardType/:accountId/:functionId/:invocations"
```
Comment (mrmckeb) [inline src/containers/App/App.js]: Do we need the account ID in here? Can one user have two account IDs?

File: src/components/List/List.js (hunk 2 above)
Comment (KateVuilova) [reply src/components/List/List.js]: Done

File: src/components/Header/Header.js (hunk 1 above)
Comment (KateVuilova) [reply src/components/Header/Header.js]: We will try this approach

File: src/containers/App/App.js (hunk 3 above)
Comment (KateVuilova) [reply src/containers/App/App.js]: Yes, we need it)

**Pull request merged**